app/Controllers/importExportController.php

```diff
@@ -29,7 +29,24 @@ public function indexAction() {
 		Minz_View::prependTitle(_t('sub.import_export.title') . ' · ');
 	}
 
+	private static function megabytes($size_str) {
+		switch (substr($size_str, -1)) {
+			case 'M': case 'm': return (int)$size_str;
+			case 'K': case 'k': return (int)$size_str / 1024;
+			case 'G': case 'g': return (int)$size_str * 1024;
```

Comment:
This is the biggest type it supports? No terabytes or something? (Admittedly less likely for RAM.) :-)

Reply:
Apparently, according to https://php.net/function.ini-get